### PR TITLE
feat(toastui): upgrade toast ui and integration

### PR DESCRIPTION
**Pull request by kwilson-bits**
Upgrade Toast UI from v1.4.6 to v2.5.X and make necessary changes to how we alter the settings and extract values out of the Toast Editor.

### CODE REVIEW

File: scss/os/_overrides_tuieditor.scss [hunk 1]
```diff
@@ -102,18 +102,121 @@
...
+    }
+
+    h6 {
+      font-size: .875rem;
+      font-weight: normal;
+    }
+
+    @media (max-width: 1200px) {
+      h1 {
+        font-size: calc(1.34375rem + 1.125vw);
```
Comment (gregroyal) [inline scss/os/_overrides_tuieditor.scss:166]: These are some very specific values. 1.34375 ?

File: scss/os/_overrides_tuieditor.scss [hunk 2]
```diff
@@ -499,10 +602,41 @@
...
 .tui-tooltip {
   z-index: $zindex-tooltip;
 }
+
+.tui-md-code {
+  background: none;
+}
+
+.tui-md-code.tui-md-marked-text {
+  color: #f89406;
```
Comment (gregroyal) [inline scss/os/_overrides_tuieditor.scss:625]: prefer variables over strings for colors

File: scss/os/_overrides_tuieditor.scss (hunk 1 above)
Comment (kwilson-bits) [reply scss/os/_overrides_tuieditor.scss:163]: I couldn't find a variable or calculation for any of the header sizes, and actually ended up grepping them out of the `.build/themes` files to find what was used. They match what was compiled before the Toast UI upgrade. The problem I ran into is that the updated Toast UI has a newer stylesheet that mucks with a lot of our theming.

File: src/os/ui/text/tuieditorlang.js [hunk 3]
```diff
@@ -8,24 +8,22 @@ goog.provide('os.ui.text.TuiEditorLang');
...
-    // options['Insert CodeBlock'] = options['Insert CodeBlock'] + ' (shift + ctrl + p)';
-    options['Italic'] = options['Italic'] + ' (ctrl + i)';
-    options['Line'] = options['Line'] + ' (ctrl + l)';
-    options['Ordered list'] = options['Ordered list'] + ' (ctrl + o)';
-    options['Strike'] = options['Strike'] + ' (ctrl + s)';
-    options['Task'] = options['Task'] + ' (alt + t)';
-    options['Unordered list'] = options['Unordered list'] + ' (ctrl + u)';
+    languageOverrides['Bold'] = 'Bold (ctrl + b)';
+    languageOverrides['Blockquote'] = 'Blockquote (alt + q)';
+    languageOverrides['Code'] = 'Inline code (shift + ctrl + c)';
```
Comment (csgibson) [inline src/os/ui/text/tuieditorlang.js:19]: Do we need an override for the code block too?
I know it was commented out in the old version but may be fixed in the new version? idk

File: scss/os/_overrides_tuieditor.scss (hunk 2 above)
Comment (kwilson-bits) [reply scss/os/_overrides_tuieditor.scss]: Color codes have been replaced with variables.

File: src/os/ui/text/tuieditorlang.js (hunk 3 above)
Comment (kwilson-bits) [reply src/os/ui/text/tuieditorlang.js:19]: It hasn't changed for the new version, so I would leave the language override out. While the shortcut still works in Chrome it still does _not_ work in Firefox.

**Review by csgibson — APPROVED** [review]

**Pull request merged**